kubernetes/loculus/values.yaml

```diff
@@ -26,7 +26,12 @@ dataUseTerms:
     open: https://#TODO-MVP/open
     restricted: https://#TODO-MVP/restricted
 s3:
-  enabled: false
+  enabled: true
+  bucket:
+    region: us-east-1
+    bucket: loculus-preview-private
+    accessKey: "[[s3accessKey]]"
```

Comment:
🙌 
(while we're touching the code, we discussed deleting these, and same in the config-preprocessor)

Reply:
yes I remember - I wanted to do it seperately but can do it now 👍

Reply:
Done! (They were actually not referenced anymore anyways, so easy to remove!)

Reply:
(sorry, of course would have been happy for it to be done separately)